modules/nodes-pods-autoscaling-policies.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-policies_{context}"]
+= Scaling policies
+
+The `autoscaling/v2beta2` API allows you to add _scaling policies_ to a horizontal pod autoscaler. A scaling policy controls how the {product-title} horizontal pod autoscaler (HPA) scales pods. Scaling policies allow you to restrict the rate that HPAs scale pods up or down by setting a specific number or specific percentage to scale in a specified period of time. You can also define a _stabilization window_ which uses previously computed desired states to control scaling if the metrics are fluctuating. You can create multiple policies for the same scaling direction, and determine which policy is used, based on the amount of change. You can also restrict the scaling by timed iterations. The HPA scales pods during an iteration, then performs scaling, as needed, in further iterations. 
+
+.Sample HPA object with a scaling policy
+[source, yaml]
+----
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: hpa-resource-metrics-memory
+  namespace: default
+spec:
+  behavior:
+    scaleDown: <1>
+      policies: <2>
+      - type: Pods <3>
+        value: 4 <4>
+        periodSeconds: 60 <5>
+      - type: Percent
+        value: 10 <6>
+        periodSeconds: 60
+      selectPolicy: Min <7>
+      stabilizationWindowSeconds: 300 <8>
+    scaleUp: <9>
+      policies:
+      - type: Pods
+        value: 5 <10>
+        periodSeconds: 70
+      - type: Percent
+        value: 12 <11>
+        periodSeconds: 80
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+...
+----
+<1> Specifies the direction for the scaling policy, either `scaleDown` or `scaleUp`. This example creates a policy for scaling down.
+<2> Defines the scaling policy. 
+<3> The `type` parameter determines if the policy scales by a specific number of pods or a percentage of pods during each iteration. The default value is `pods`.
+<4> The `value` parameter determines the amount of scaling, either the number of pods or percentage of pods during each iteration. There is no default value for scaling down by number of pods.
+<5> The `periodSeconds` parameter determines the length of a scaling iteration. The default value is `15` seconds.
+<6> The default value for scaling down by percentage is 100%.
+<7>The `selectPolicy` parameter determines which policy to use first, if multiple policies are defined. Specify `Max` to use the policy that allows the highest amount of change, `Min` to use the policy that allows the lowest amount of change, or `Disabled` to prevent the HPA from scaling in that policy direction. The default value is `Max`.
```

Comment:
This callout needs a space after it: `<7> The`. It is not rendering correctly because of this.

Reply:
Thank you. I noticed that earlier but forgot to fix it!